### ISSUE TITLE
Login failed for user <blank>

### ISSUE THREAD

**Issue by aprilmintacpineda** [sequelize/sequelize-auto#387]
```js
const SequelizeAuto = require('sequelize-auto');
const config = require('../src/config');

console.log(config); // is correct

const sequelizeAuto = new SequelizeAuto(
  config.DB_NAME,
  config.DB_USER,
  config.DB_PASS,
  {
    host: config.DB_HOST,
    dialect: 'mssql',
    directory: require('path').join(__dirname, '../src/models')
  }
);

sequelizeAuto.run(err => {
  if (err) throw err;
  console.log(sequelizeAuto); // I could see the `config` here is showing the correct values
});
```

Is throw this error

```
name: 'SequelizeConnectionError',
  message: "Login failed for user ''.",
  parent: ConnectionError: Login failed for user ''.
```

However, on the `.run` callback, the `err` is `null`.

I expect the models to be edited accordingly but nothing happens when I run the script.

**Comment by r3d12** [comment]
https://github.com/sequelize/sequelize-auto/issues/360#issuecomment-486515484

That did it for me 

make sure you have run sequelize init
and edit the config with the appropriate data (you can remove everything it adds after your done) 

1. go to global tedios module if you installed sequelize-auto with the -g commang
     C:\Users\username\AppData\Roaming\npm\node_modules\tedious\lib\connection.js

     if you installed sequelize-auto locally(in your web app folder) got to your local tedios module and make the changes

2.  look for chunk of code

 if (config.authentication !== undefined) {
      if (typeof config.authentication !== 'object' || config.authentication === null) {
        throw new TypeError('The "config.authentication" property must be of type Object.');
      } 

3. add this above it and below "let authentication"

       // Insert the following authentication object
        config.authentication = {
          type: "default"
        };

4. look for 
        authentication = {
          type: 'default',
          options: {
            userName: options.userName,
            password: options.password
          }
        };
5. changer username and password to 
            userName: config.userName,
            password: config.password

then run your command

**Comment by soryy708** [comment]
Duplicate of https://github.com/sequelize/sequelize-auto/issues/360

**Comment by steveschmitt** [comment]
I think this is fixed in `sequelize-auto@0.5.0`  
Please try it and close this issue if so.  Thanks!

**Comment by zafaralam** [comment]
> I think this is fixed in `sequelize-auto@0.5.0`
> Please try it and close this issue if so. Thanks!

The issue with login seems to be resolved. However there is an issue with the code generated.

The primary key/identity column objects with the generated file is missing a comma after the 'autoIncrement' property and before the 'comment' property.

An example of one of the models generated with sequelize-auto@0.5.0. I'm using mssql dialect.

```javascript
/* jshint indent: 1 */

module.exports = function(sequelize, DataTypes) {
	return sequelize.define('areaTargets', {
		'id': {
			type: DataTypes.INTEGER,
			allowNull: false,
			primaryKey: true,
			autoIncrement: true			comment: '',
			field: 'id'
		},
		'createdBy': {
			type: DataTypes.STRING(255),
			allowNull: false,
			defaultValue: "admin",
			comment: '',
			field: 'created_by'
		},
		'updatedBy': {
			type: DataTypes.STRING(255),
			allowNull: false,
			defaultValue: "admin",
			comment: '',
			field: 'updated_by'
		},
		'createdAt': {
			type: DataTypes.DATE,
			allowNull: false,
			defaultValue: "(getdate())",
			comment: '',
			field: 'created_at'
		},
		'updatedAt': {
			type: DataTypes.DATE,
			allowNull: false,
			defaultValue: "(getdate())",
			comment: '',
			field: 'updated_at'
		},
		'properties': {
			type: DataTypes.STRING,
			allowNull: true,
			defaultValue: "{}",
			comment: '',
			field: 'properties'
		},
		'areaTypeId': {
			type: DataTypes.INTEGER,
			allowNull: true,
			comment: '',
			field: 'area_type_id'
		},
		'targetId': {
			type: DataTypes.INTEGER,
			allowNull: true,
			comment: '',
			field: 'target_id'
		},
		'areaName': {
			type: DataTypes.STRING(1000),
			allowNull: false,
			comment: '',
			field: 'area_name'
		},
		'targetValue': {
			type: DataTypes.DOUBLE,
			allowNull: false,
			defaultValue: "((0))",
			comment: '',
			field: 'target_value'
		},
		'targetMetCriteriaId': {
			type: DataTypes.INTEGER,
			allowNull: true,
			comment: '',
			field: 'target_met_criteria_id'
		},
		'validFrom': {
			type: DataTypes.DATE,
			allowNull: false,
			comment: '',
			field: 'valid_from'
		},
		'validUntil': {
			type: DataTypes.DATE,
			allowNull: false,
			comment: '',
			field: 'valid_until'
		}
	}, {
		sequelize,
		tableName: 'AreaTargets'
	});
};
```

**Comment by steveschmitt** [comment]
Good catch!  Thanks!  I'll fix it in the next release.
Thanks!
